### PR TITLE
DTSPB-2704: Upgrade Springboot version 2.6.6 for probate services

### PR DESCRIPTION
**Pull request by srochani**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by sanjayparekh-hmcts — APPROVED** [review]

File: build.gradle [hunk 1]
```diff
@@ -151,7 +151,7 @@ def versions = [
         springCloudStarterNetflixHystrix: '2.2.10.RELEASE',
         serenity                        : '2.6.0',
         serenityreporter                : '2.6.0',
-        springBootVersion               : '2.6.3',
+        springBootVersion               : '2.6.6',
```
Comment (isha2040) [inline build.gradle:154]: hi Sunil, looks like https://github.com/hmcts/probate-back-office/blob/f589a24d64c37c573b4dfe5ca41b612398c46094/build.gradle#L17 still points to 2.6.3 . Could you correct that too please.

**Comment by srochani** [comment]
Closing and reopening the PR

**Comment by srochani** [comment]
reopening the PR

**Comment by srochani** [comment]
Closing and opening new PRs

**Pull request closed without merging**